### PR TITLE
Handle the case when there is no info for payments and addresses

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Handle the case when there is no info for payments and addresses

#### How should this be manually tested?
[workspace](https://felipe--recorrenciaqa.myvtex.com/Account/#/addresses)

image

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EmptyPlaceholder.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const EmptyPlaceholder = ({ children }) => {
```
Comment (kevinch) [inline react/components/EmptyPlaceholder.js]: Hint, tem EmptyState in Styleguide.

File: react/locales/fr.json [hunk 2]
```diff
@@ -34,9 +34,11 @@
   "addresses.addAddress": "Ajouter addresse",
   "addresses.saveAddress": "Sauvegarder addresse",
   "addresses.deleteAddress": "Supprimer addresse",
+  "addresses.notFound": "Vous n'avez aucune adresse enregistré!",
```
Comment (kevinch) [inline react/locales/fr.json]: Exclamation points take a space before in french.
```suggestion
  "addresses.notFound": "Vous n'avez aucune adresse enregistré !",
```

File: react/locales/fr.json [hunk 3]
```diff
@@ -34,9 +34,11 @@
...
 
   "payments.cardNumber": "Numéro de carte",
   "payments.billingAddress": "Adresse de facturation",
+  "payments.notFound": "Vous n'avez aucun moyen de paiement enregistré!",
```
Comment (kevinch) [inline react/locales/fr.json]: Exclamation points take a space before in french.
```suggestion
  "payments.notFound": "Vous n'avez aucun moyen de paiement enregistré !",
```

File: react/components/EmptyPlaceholder.js (hunk 1 above)
Comment (salesfelipe) [reply react/components/EmptyPlaceholder.js]: thx :D

File: react/locales/fr.json (hunk 2 above)
Comment (salesfelipe) [reply react/locales/fr.json]: D'accord 👍

File: react/locales/fr.json (hunk 3 above)
Comment (salesfelipe) [reply react/locales/fr.json]: D'accord 👍

File: react/locales/pt.json [hunk 4]
```diff
@@ -34,9 +34,11 @@
   "addresses.addAddress": "Adicionar endereço",
   "addresses.saveAddress": "Salvar endereço",
   "addresses.deleteAddress": "Remover endereço",
+  "addresses.notFound": "Você não tem nenhum endereço cadastrado!",
 
   "payments.cardNumber": "Número do cartão",
   "payments.billingAddress": "Endereço de cobrança",
+  "payments.notFound": "Você não tem nenhum método de pagamento registrado!",
```
Comment (klzns) [inline react/locales/pt.json]: Parece que ta gritando por conta da exclamação, nao acha nao?

**Pull request merged**